src/shared/GameRound/GameRoundAdmin.tsx

```diff
@@ -380,12 +384,20 @@ const GameRoundAdmin = (props: any) => {
           </div>
           <div style={{ marginTop: 10, width: '100%' }}>
             <p style={{ textAlign: 'center' }}>Wrong answers</p>
-            <Button variant="outlined" onClick={() => setAllWrong()} style={{ marginRight: 10 }}>
+            <Button variant="outlined" onClick={() => {
+              setAllWrong();
+              }} 
+              style={{ marginRight: 10 }}>
               Set All wrong
             </Button>
             <Button
               variant="outlined"
-              onClick={() => setCounterWrongAnswers((prevState) => Math.min(prevState + 1, 3))}
+              onClick={() => {
+                setCounterWrongAnswers((prevState) => Math.min(prevState + 1, 3))
+                if (counterWrongAnswers <= 3){
+                  playSound(wrongAnswerSound, 5000);
```

Comment:
On the client the length the X is shown is smaller than the sound, I think it is 2000ms